server/auth.go

```diff
@@ -45,3 +45,84 @@ func AuthorizedToken(auth oauth2.Authenticator, logger chronograf.Logger, next h
 		return
 	})
 }
+
+// AuthorizedUser extracts the user name and provider from context. If the user and provider can be found on the
```

Comment:
niggling, but it looks like these comments are longer than 80 char per line so they're wrapping awkwardly ;)

Reply:
fixed 💯